Description: Add missing includes because of the new version of liborthancframework-dev
Author: Sebastien Jodogne <s.jodogne@orthanc-labs.com>
Forwarded: yes
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: OrthancGdcm-1.1/Plugin/Plugin.cpp
===================================================================
--- OrthancGdcm-1.1.orig/Plugin/Plugin.cpp
+++ OrthancGdcm-1.1/Plugin/Plugin.cpp
@@ -23,6 +23,7 @@
 
 #include <Compatibility.h>
 #include <DicomFormat/DicomMap.h>
+#include <Logging.h>
 #include <MultiThreading/Semaphore.h>
 #include <Toolbox.h>
 
